src/pages/SearchPage/Heatmap/Heatmap.style.js

```diff
@@ -15,10 +15,10 @@ const HeatmapContainer = styled.div`
   flex-direction: column;
   height: 332px;
   width: 100%;
-  max-width: 1114px;
+  width: 1114px;
   margin-top: 8px;
   @media (max-width: 1134px) {
-    width: calc(100% - 20px);
+    max-width: calc(100% - 20px);
```

Comment:
I meant that you can move this outside the media-query as well. Then the max-width will kick in once the screen is narrower than 100% - 20px

Reply:
Not sure if you still need the media query. Does `text-align` have an effect here? If so leave the media query of course